src/utils/daos_hdlr.c

```diff
@@ -1570,6 +1570,7 @@ dm_connect(struct cmd_args_s *ap,
 				DH_PERROR_DER(ap, rc, "failed to open container");
 				D_GOTO(err, rc);
 			}
+			fprintf(ap->outstream, "Successfully created container %s\n", ca->dst_cont);
```

Comment:
An alternate approach might be to do something like
```
ca->dst_cont_created = True
```
Then let the GO code conditionally print the message.